int32 sec
int32 nsec
int32 seq
bool last_msg
float32 signal
float64 time
